x-pack/plugins/observability_solution/infra/public/components/ml/anomaly_detection/anomalies_table/anomalies_table.tsx

```diff
@@ -74,11 +80,30 @@ const AnomalyActionMenu = ({
   influencerField: string;
   influencers: string[];
   disableShowInInventory?: boolean;
+  hostName?: string;
 }) => {
   const [isOpen, setIsOpen] = useState(false);
   const close = useCallback(() => setIsOpen(false), [setIsOpen]);
   const handleToggleMenu = useCallback(() => setIsOpen(!isOpen), [isOpen]);
   const { onViewChange } = useWaffleViewState();
+  const { metricsView } = useMetricsDataViewContext();
+  const location = useLocation<RouteState>();
+  const {
+    services: {
+      application: { navigateToApp },
+      data: {
+        query: { filterManager: filterManagerService },
+      },
+    },
+  } = useKibanaContextForPlugin();
+
+  const addHostFilter = useCallback(
```

Comment:
Q: Do we need `useCallback` here? I see we use the function in a `useCallback` already so I am wondering if it's necessary here.

Reply:
Agree. Thanks for pointing it out. Updated :)